diff --git a/bumblebee-3.2.1/src/bbsecondary.c b/bumblebee-3.2.1/src/bbsecondary.c
--- a/bumblebee-3.2.1/src/bbsecondary.c
+++ b/bumblebee-3.2.1/src/bbsecondary.c
@@ -138,7 +138,7 @@
   if (!bb_is_running(bb_status.x_pid)) {
     char pci_id[12];
     static char *x_conf_file;
-    snprintf(pci_id, 12, "PCI:%02x:%02x:%o", pci_bus_id_discrete->bus,
+    snprintf(pci_id, 12, "PCI:%02d:%02d:%o", pci_bus_id_discrete->bus,
             pci_bus_id_discrete->slot, pci_bus_id_discrete->func);
     if (!x_conf_file) {
       x_conf_file = xorg_path_w_driver(bb_config.x_conf_file, bb_config.driver);
diff --git a/bumblebee-3.2.1/src/module.c b/bumblebee-3.2.1/src/module.c
--- a/bumblebee-3.2.1/src/module.c
+++ b/bumblebee-3.2.1/src/module.c
@@ -92,11 +92,15 @@
  * @return 1 if the driver is succesfully unloaded, 0 otherwise
  */
 int module_unload(char *driver) {
-  if (module_is_loaded(driver) == 1) {
+
+  if (module_is_loaded(driver) == 1 &&
+            module_is_loaded("nvidia_modeset") == 1) {
     int retries = 30;
     bb_log(LOG_INFO, "Unloading %s driver\n", driver);
     char *mod_argv[] = {
-      "rmmod",
+      "modprobe",
+      "-r",
+      "nvidia-modeset",
       driver,
       NULL
     };
@@ -109,6 +113,24 @@
       return 0;
     }
   }
+    else if (module_is_loaded(driver) == 1) {
+      int retries = 30;
+      bb_log(LOG_INFO, "Unloading %s driver\n", driver);
+      char *mod_argv[] = {
+        "modprobe",
+        "-r",
+        driver,
+        NULL
+      };
+      bb_run_fork_wait(mod_argv, 10);
+      while (retries-- > 0 && module_is_loaded(driver) == 1) {
+        usleep(100000);
+      }
+      if (module_is_loaded(driver) == 1) {
+        bb_log(LOG_ERR, "Unloading %s driver timed out.\n", driver);
+        return 0;
+      }
+    }
   return 1;
 }
 