diff -rupN os-prober.old/common.sh os-prober/common.sh
--- os-prober.old/common.sh	2015-07-30 15:47:15.493728925 +0200
+++ os-prober/common.sh	2015-07-30 16:02:46.028549186 +0200
@@ -116,6 +116,29 @@ fs_type () {
 	fi
 }
 
+is_dos_extended_partition() {
+	if type blkid >/dev/null 2>&1; then
+		local output
+
+		output="$(blkid -o export $1)"
+
+		# old blkid (util-linux << 2.24) errors out on extended p.
+		if [ "$?" = "2" ]; then
+			return 0
+		fi
+
+		# dos partition type and no filesystem type?...
+		if echo $output | grep -q ' PTTYPE=dos ' &&
+				! echo $output | grep -q ' TYPE='; then
+			return 0
+		else
+			return 1
+		fi
+	fi
+
+	return 1
+}
+
 parse_proc_mounts () {
 	while read -r line; do
 		set -f
diff -rupN os-prober.old/linux-boot-probes/common/50mounted-tests os-prober/linux-boot-probes/common/50mounted-tests
--- os-prober.old/linux-boot-probes/common/50mounted-tests	2015-07-30 15:47:15.487062200 +0200
+++ os-prober/linux-boot-probes/common/50mounted-tests	2015-07-30 16:05:47.293461931 +0200
@@ -20,6 +20,10 @@ elif [ "$types" = ntfs ]; then
 		types='ntfs-3g ntfs'
 	fi
 elif [ -z "$types" ]; then
+	if is_dos_extended_partition "$partition"; then
+		debug "$1 looks like an extended dos partition; skipping"
+		exit 0
+	fi
 	if type cryptsetup >/dev/null 2>&1 && \
 	   cryptsetup luksDump "$partition" >/dev/null 2>&1; then
 		debug "$1 is a LUKS partition; skipping"
diff -rupN os-prober.old/linux-boot-probes/mounted/common/40grub2 os-prober/linux-boot-probes/mounted/common/40grub2
--- os-prober.old/linux-boot-probes/mounted/common/40grub2	2015-07-30 15:47:15.487062200 +0200
+++ os-prober/linux-boot-probes/mounted/common/40grub2	2015-07-30 15:53:39.593770984 +0200
@@ -78,10 +78,10 @@ parse_grub_menu () {
 				fi
 			;;
 			initrd)
-				initrd="$(echo "$2" | sed 's/(.*)//')"
-				# Initrd same.
 				if [ "$partition" != "$bootpart" ]; then
-					initrd="/boot$initrd"
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s|/|/boot/|g;s/(.*)//')"
+				else
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s/(.*)//')"
 				fi
 			;;
 			"}")
diff -rupN os-prober.old/linux-boot-probes/mounted/x86/40grub os-prober/linux-boot-probes/mounted/x86/40grub
--- os-prober.old/linux-boot-probes/mounted/x86/40grub	2015-07-30 15:47:15.487062200 +0200
+++ os-prober/linux-boot-probes/mounted/x86/40grub	2015-07-30 15:57:00.508864273 +0200
@@ -66,10 +66,10 @@ parse_grub_menu () {
 			initrd)
 				# Hack alert take 2: sed off any (hdn,n)
 				# See #566102
-				initrd="$(echo "$2" | sed 's/(.*)//')"
-				# Initrd same.
 				if [ "$partition" != "$bootpart" ]; then
-					initrd="/boot$initrd"
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s|/|/boot/|g;s/(.*)//')"
+				else
+					initrd="$(echo "$line" | sed 's/initrd//;s/\t//;s/(.*)//')"
 				fi
 			;;
 			boot)
diff -rupN os-prober.old/os-probes/common/50mounted-tests os-prober/os-probes/common/50mounted-tests
--- os-prober.old/os-probes/common/50mounted-tests	2015-07-30 15:47:15.490395562 +0200
+++ os-prober/os-probes/common/50mounted-tests	2015-07-30 16:07:01.540774778 +0200
@@ -20,6 +20,10 @@ elif [ "$types" = ntfs ]; then
 		types='ntfs-3g ntfs'
 	fi
 elif [ -z "$types" ]; then
+	if is_dos_extended_partition "$partition"; then
+		debug "$1 looks like an extended dos partition; skipping"
+		exit 0
+	fi
 	if type cryptsetup >/dev/null 2>&1 && \
 	   cryptsetup luksDump "$partition" >/dev/null 2>&1; then
 		debug "$1 is a LUKS partition; skipping"
diff -rupN os-prober.old/os-probes/mounted/x86/20microsoft os-prober/os-probes/mounted/x86/20microsoft
--- os-prober.old/os-probes/mounted/x86/20microsoft	2015-07-30 15:47:15.493728925 +0200
+++ os-prober/os-probes/mounted/x86/20microsoft	2015-07-30 15:59:32.140190102 +0200
@@ -31,7 +31,9 @@ if item_in_dir -q bootmgr "$2"; then
 	for boot in $(item_in_dir boot "$2"); do
 		bcd=$(item_in_dir bcd "$2/$boot")
 		if [ -n "$bcd" ]; then
-			if grep -aqs "W.i.n.d.o.w.s. .8" "$2/$boot/$bcd"; then
+			if grep -aqs "W.i.n.d.o.w.s. .10" "$2/$boot/$bcd"; then
+				long="Windows 10 (loader)"
+			elif grep -aqs "W.i.n.d.o.w.s. .8" "$2/$boot/$bcd"; then
 				long="Windows 8 (loader)"
 			elif grep -aqs "W.i.n.d.o.w.s. .7" "$2/$boot/$bcd"; then
 				long="Windows 7 (loader)"
