diff --git a/nv-drm.c b/nv-drm.c
index ba9fa61..bf7a177 100644
--- a/nv-drm.c
+++ b/nv-drm.c
@@ -71,7 +71,11 @@ static const struct file_operations nv_drm_fops = {
 };
 
 static struct drm_driver nv_drm_driver = {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 9, 0)
+    .driver_features = DRIVER_LEGACY,
+#else
     .driver_features = 0,
+#endif
     .load = nv_drm_load,
     .unload = nv_drm_unload,
     .fops = &nv_drm_fops,
-- 
2.7.4
