--- a/steam	2016-09-17 21:44:03.000000000 +0200
+++ b/steam	2016-09-17 21:49:11.000000000 +0200
@@ -9,6 +9,9 @@
 # Get the full name of this script
 export STEAMSCRIPT="$(cd "${0%/*}" && echo "$PWD")/${0##*/}"
 
+# preload some libs, so we don't need to remove them on mesa-only systems
+export LD_PRELOAD='/usr/$LIB/libstdc++.so.6 /usr/$LIB/libgcc_s.so.1 /usr/$LIB/libxcb.so.1 /usr/$LIB/libgpg-error.so'
+
 # When this version is incremented a new package will be deployed to:
 #	http://repo.steampowered.com/steam
 # and an e-mail will be sent to the steam package update mailing list:

