--- a/src/r8168_n.c	2016-06-04 16:12:05.746880000 +0200
+++ b/src/r8168_n.c	2016-06-04 16:27:30.956040227 +0200
@@ -24559,7 +24559,11 @@ rtl8168_start_xmit(struct sk_buff *skb,
         wmb();
         txd->opts1 = cpu_to_le32(opts1);
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
+        netif_trans_update(dev);
+#else
         dev->trans_start = jiffies;
+#endif
 
         tp->cur_tx += frags + 1;
 
