From 2951723d0d708fde4c26ec22eddea86523dad8ae Mon Sep 17 00:00:00 2001
From: udeved <artoo@manjaro.org>
Date: Wed, 23 Dec 2015 21:56:13 +0100
Subject: [PATCH] forgot to decalte method getLXQTSU

---
 src/wmhelper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/wmhelper.h b/src/wmhelper.h
index de112e0..b92fd36 100644
--- a/src/wmhelper.h
+++ b/src/wmhelper.h
@@ -112,6 +112,7 @@ public:
   static QString getKDESUCommand();
   static QString getTDESUCommand();
   static QString getGKSUCommand();
+  static QString getLXQTSUCommand();
   static QString getXFCEEditor();
   static QString getSUCommand();
   static void openFile(const QString& fileName); //fileName is Path + Name
-- 
2.6.4

