diff -Npur vboxhost.orig/5.1.8_OSE/build/vboxdrv/r0drv/linux/memobj-r0drv-linux.c vboxhost/5.1.8_OSE/build/vboxdrv/r0drv/linux/memobj-r0drv-linux.c
--- vboxhost.orig/5.1.8_OSE/build/vboxdrv/r0drv/linux/memobj-r0drv-linux.c	2016-10-30 13:28:21.381588000 +0100
+++ vboxhost/5.1.8_OSE/build/vboxdrv/r0drv/linux/memobj-r0drv-linux.c	2016-10-30 13:49:00.963252343 +0100
@@ -1050,7 +1050,9 @@ DECLHIDDEN(int) rtR0MemObjNativeLockUser(PPRTR0MEMOBJINTERNAL ppMem, RTR3PTR R3P
             rc = get_user_pages(R3Ptr,                  /* Where from. */
                                 cPages,                 /* How many pages. */
                                 fWrite,                 /* Write to memory. */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 9, 0)
                                 fWrite,                 /* force write access. */
+#endif /* LINUX_VERSION_CODE < KERNEL_VERSION(4, 9, 0) */
                                 &pMemLnx->apPages[0],   /* Page array. */
                                 papVMAs);               /* vmas */
         /*
@@ -1064,7 +1066,9 @@ DECLHIDDEN(int) rtR0MemObjNativeLockUser(PPRTR0MEMOBJINTERNAL ppMem, RTR3PTR R3P
                                 R3Ptr,                  /* Where from. */
                                 cPages,                 /* How many pages. */
                                 fWrite,                 /* Write to memory. */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 9, 0)
                                 fWrite,                 /* force write access. */
+#endif /* LINUX_VERSION_CODE < KERNEL_VERSION(4, 9, 0) */
                                 &pMemLnx->apPages[0],   /* Page array. */
                                 papVMAs);               /* vmas */
 #else /* LINUX_VERSION_CODE < KERNEL_VERSION(4, 6, 0) */
