diff -Naur a/src/placement.c b/src/placement.c
--- a/src/placement.c	2015-03-01 06:17:44.000000000 +1100
+++ b/src/placement.c	2015-03-06 01:14:19.375259636 +1100
@@ -734,6 +734,9 @@
 
     c->x = best_x;
     c->y = best_y;
+    
+    FLAG_SET (c->flags, CLIENT_FLAG_MAXIMIZED_HORIZ);
+    FLAG_SET (c->flags, CLIENT_FLAG_MAXIMIZED_VERT);
 }
 
 static void
@@ -744,6 +747,9 @@
 
     c->x = MAX (full_x + frameExtentLeft(c) + (full_w - frameExtentWidth(c)) / 2, full_x + frameExtentLeft(c));
     c->y = MAX (full_y + frameExtentTop(c) + (full_h - frameExtentHeight(c)) / 2, full_y + frameExtentTop(c));
+    
+    FLAG_SET (c->flags, CLIENT_FLAG_MAXIMIZED_HORIZ);
+    FLAG_SET (c->flags, CLIENT_FLAG_MAXIMIZED_VERT);
 }
 
 static void
@@ -760,6 +766,9 @@
 
     c->x = MAX (c->x, full_x + frameExtentLeft(c));
     c->y = MAX (c->y, full_y + frameExtentTop(c));
+    
+    FLAG_SET (c->flags, CLIENT_FLAG_MAXIMIZED_HORIZ);
+    FLAG_SET (c->flags, CLIENT_FLAG_MAXIMIZED_VERT);
 }
 
 void
