diff -ru --no-dereference a/overrides/mips/com.deepin.dde.dock.override b/overrides/mips/com.deepin.dde.dock.override
--- a/overrides/mips/com.deepin.dde.dock.override	2016-09-04 19:06:09.918990868 +0200
+++ b/overrides/mips/com.deepin.dde.dock.override	2016-09-03 19:27:55.026379607 +0200
@@ -1,3 +1,3 @@
 [com.deepin.dde.dock]
-docked-apps=['deepin-toggle-desktop', 'deepin-wm-multitaskingview','org.gnome.nautilus','audacious-qt','smplayer','firefox','wps-office-wps','wps-office-wpp','wps-office-et','dde-control-center']
-hide-mode='keep-showing'
+docked-apps=['dde-file-manager','chromium','deepin-terminal','deepin-toggle-desktop','deepin-wm-multitaskingview','dde-control-center']
+hide-mode='smart-hide'
diff -ru --no-dereference a/overrides/mips/com.deepin.wrap.gnome.desktop.override b/overrides/mips/com.deepin.wrap.gnome.desktop.override
--- a/overrides/mips/com.deepin.wrap.gnome.desktop.override	2016-09-04 19:06:09.918990868 +0200
+++ b/overrides/mips/com.deepin.wrap.gnome.desktop.override	2016-09-03 19:27:55.026379607 +0200
@@ -13,7 +13,7 @@
 mouse-button-modifier=''
 
 [com.deepin.wrap.gnome.desktop.interface]
-cursor-theme='deepin'
+cursor-theme='Breeze'
 gtk-key-theme='Default'
 gtk-theme='deepin'
 icon-theme='deepin'
diff -ru --no-dereference a/overrides/mips/other.override b/overrides/mips/other.override
--- a/overrides/mips/other.override	2016-09-04 19:06:09.918990868 +0200
+++ b/overrides/mips/other.override	2016-09-03 19:27:55.026379607 +0200
@@ -4,3 +4,17 @@
 [org.gnome.nautilus.preferences]
 executable-text-activation='ask'
 sort-directories-first=true
+
+[org.gnome.nautilus.icon-view]
+default-zoom-level='small'
+
+[org.gnome.Terminal.Legacy.Profile]
+allow-bold=true
+background-color='rgb(69,70,73)'
+use-theme-colors=false
+font='Droid Sans Mono Regular 11'
+foreground-color='rgb(220,233,235)'
+palette=['rgb(0,0,0)', 'rgb(170,0,0)', 'rgb(0,170,0)', 'rgb(170,85,0)', 'rgb(0,0,170)', 'rgb(170,0,170)', 'rgb(0,170,170)', 'rgb(170,170,170)', 'rgb(85,85,85)', 'rgb(255,85,85)', 'rgb(85,255,85)', 'rgb(255,255,85)', 'rgb(85,85,255)', 'rgb(255,85,255)', 'rgb(85,255,255)', 'rgb(255,255,255)']
+rewrap-on-resize=true
+visible-name='manjaro'
+
diff -ru --no-dereference a/overrides/x86/com.deepin.dde.dock.override b/overrides/x86/com.deepin.dde.dock.override
--- a/overrides/x86/com.deepin.dde.dock.override	2016-09-04 19:06:09.918990868 +0200
+++ b/overrides/x86/com.deepin.dde.dock.override	2016-09-03 19:27:55.026379607 +0200
@@ -1,3 +1,3 @@
 [com.deepin.dde.dock]
-docked-apps=['deepin-toggle-desktop', 'deepin-wm-multitaskingview','dde-file-manager','deepin-appstore','deepin-music-player','deepin-movie','google-chrome','dde-control-center']
-hide-mode='keep-showing'
+docked-apps=['dde-file-manager','chromium','deepin-terminal','deepin-toggle-desktop','deepin-wm-multitaskingview','dde-control-center']
+hide-mode='smart-hide'
diff -ru --no-dereference a/overrides/x86/com.deepin.wrap.gnome.desktop.override b/overrides/x86/com.deepin.wrap.gnome.desktop.override
--- a/overrides/x86/com.deepin.wrap.gnome.desktop.override	2016-09-04 19:06:09.918990868 +0200
+++ b/overrides/x86/com.deepin.wrap.gnome.desktop.override	2016-09-03 19:27:55.026379607 +0200
@@ -13,7 +13,7 @@
 mouse-button-modifier=''
 
 [com.deepin.wrap.gnome.desktop.interface]
-cursor-theme='deepin'
+cursor-theme='Breeze'
 gtk-key-theme='Default'
 gtk-theme='deepin'
 icon-theme='deepin'
diff -ru --no-dereference a/overrides/x86/other.override b/overrides/x86/other.override
--- a/overrides/x86/other.override	2016-09-04 19:06:09.918990868 +0200
+++ b/overrides/x86/other.override	2016-09-03 19:27:55.026379607 +0200
@@ -4,3 +4,16 @@
 [org.gnome.nautilus.preferences]
 executable-text-activation='ask'
 sort-directories-first=true
+
+[org.gnome.nautilus.icon-view]
+default-zoom-level='small'
+
+[org.gnome.Terminal.Legacy.Profile]
+allow-bold=true
+background-color='rgb(69,70,73)'
+use-theme-colors=false
+font='Droid Sans Mono Regular 11'
+foreground-color='rgb(220,233,235)'
+palette=['rgb(0,0,0)', 'rgb(170,0,0)', 'rgb(0,170,0)', 'rgb(170,85,0)', 'rgb(0,0,170)', 'rgb(170,0,170)', 'rgb(0,170,170)', 'rgb(170,170,170)', 'rgb(85,85,85)', 'rgb(255,85,85)', 'rgb(85,255,85)', 'rgb(255,255,85)', 'rgb(85,85,255)', 'rgb(255,85,255)', 'rgb(85,255,255)', 'rgb(255,255,255)']
+rewrap-on-resize=true
+visible-name='manjaro'
diff -ru --no-dereference a/schemas/com.deepin.dde.appearance.gschema.xml b/schemas/com.deepin.dde.appearance.gschema.xml
--- a/schemas/com.deepin.dde.appearance.gschema.xml	2016-09-04 19:06:09.918990868 +0200
+++ b/schemas/com.deepin.dde.appearance.gschema.xml	2016-09-04 19:03:58.311280183 +0200
@@ -16,7 +16,7 @@
       </description>
     </key>
     <key type="s" name="cursor-theme">
-      <default>'deepin'</default>
+      <default>'Breeze'</default>
       <summary>Cursor theme</summary>
       <description>
         Cursor theme name. Used only by Xservers that support the Xcursor extension.
diff -ru --no-dereference a/schemas/com.deepin.xsettings.gschema.xml b/schemas/com.deepin.xsettings.gschema.xml
--- a/schemas/com.deepin.xsettings.gschema.xml	2016-09-04 19:06:09.922324228 +0200
+++ b/schemas/com.deepin.xsettings.gschema.xml	2016-09-03 19:27:55.029712985 +0200
@@ -35,14 +35,14 @@
       </description>
     </key>
     <key type="s" name="gtk-cursor-theme-name">
-      <default>'deepin'</default>
+      <default>'Breeze'</default>
       <summary>Cursor theme</summary>
       <description>
         Cursor theme name. Used only by Xservers that support the Xcursor extension.
       </description>
     </key>
     <key type="s" name="gtk-font-name">
-      <default>'sans-serif 10'</default>
+      <default>'Noto Sans 10'</default>
       <summary>Default font</summary>
       <description>
         Name of the default font used by gtk+.

