--- a/driver/wrapndis.c	2013-11-28 20:42:36.000000000 +0100
+++ b/driver/wrapndis.c	2016-06-04 15:50:05.235080153 +0200
@@ -704,7 +704,11 @@ static void tx_worker(struct work_struct
 			n = wnd->max_tx_packets;
 		n = mp_tx_packets(wnd, wnd->tx_ring_start, n);
 		if (n) {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
+			netif_trans_update(wnd->net_dev);
+#else
 			wnd->net_dev->trans_start = jiffies;
+#endif
 			wnd->tx_ring_start =
 				(wnd->tx_ring_start + n) % TX_RING_SIZE;
 			wnd->is_tx_ring_full = 0;
