--- mate-tweak	2017-07-20 00:25:42.000000000 +0300
+++ mate-tweak.new	2017-08-01 16:46:58.363141000 +0300
@@ -480,15 +480,6 @@
             if metacity_schema:
                 self.set_string('org.gnome.metacity.theme', None, 'name', mate_theme)
 
-        # Work around an issue what a transparent panel will crash when
-        # the compsitor is changed.
-        # TODO: Do this for all panels regardless of layout used and
-        # and only flip solid/transparent where required.
-        if self.current_layout == 'mutiny':
-            background_type = self.get_dconf_value('/org/mate/panel/toplevels/left/background/type')
-            background_color = self.get_dconf_value('/org/mate/panel/toplevels/left/background/color')
-            self.set_dconf_value('/org/mate/panel/toplevels/left/background/type',"'none'")
-
         # If switching away from a compton or xcompmgr composited window
         # manager kill compton.
         if 'compton' in self.current_wm:
@@ -501,11 +492,6 @@
         else:
             subprocess.Popen([new_wm, '--replace', wm_params], stdout=DEVNULL, stderr=DEVNULL)
 
-        if self.current_layout == 'mutiny':
-            time.sleep(0.5)
-            self.set_dconf_value('/org/mate/panel/toplevels/left/background/type', background_type)
-            self.set_dconf_value('/org/mate/panel/toplevels/left/background/color', background_color)
-
         # Update current_wm to the new one.
         self.previous_wm = self.current_wm
         self.current_wm = new_wm
@@ -1052,9 +1038,10 @@
         if os.path.exists('/usr/lib/mate-menu/mate-menu.py'):
             self.mate_menu_available = True
 
-        if os.path.exists('/usr/lib/' + self.multiarch + '/brisk-menu/brisk-menu') and \
-            os.path.exists('/usr/share/mate-panel/applets/com.solus_project.brisk.BriskMenu.mate-panel-applet'):
-            self.brisk_menu_available = True
+        if os.path.exists('/usr/lib/' + self.multiarch + '/brisk-menu/brisk-menu') or \
+            os.path.exists('/usr/lib/' + '/brisk-menu/brisk-menu'):
+            if os.path.exists('/usr/share/mate-panel/applets/com.solus_project.brisk.BriskMenu.mate-panel-applet'):
+             self.brisk_menu_available = True
 
         if os.path.exists('/usr/lib/mate-panel/appmenu-mate') and \
             os.path.exists('/usr/share/mate-panel/applets/org.vala-panel.appmenu.mate-panel-applet'):
@@ -1181,6 +1168,32 @@
         if self.panel_layout_exists('ubuntu-mate'):
             panels.append([_("Traditional"), "ubuntu-mate"])
 
+        if self.panel_layout_exists('manjaro') and \
+           self.brisk_menu_available:
+            panels.append([_("Manjaro"), "manjaro"])
+
+        if self.panel_layout_exists('redmond-no-indicators') and \
+           not self.indicators_available:
+            panels.append([_("Redmond (no indicators)"), "redmond-no-indicators"])
+
+        if self.panel_layout_exists('netbook-no-indicators') and \
+           self.maximus_available and \
+           not self.indicators_available:
+            panels.append([_("Netbook (no indicators)"), "netbook-no-indicators"])
+
+        if self.panel_layout_exists('mutiny-no-indicators') and \
+           self.mate_dock_available and \
+           self.appmenu_applet_available and \
+           not self.indicators_available:
+            panels.append([_("Mutiny (no indicators)"), "mutiny-no-indicators"])
+
+        if self.dock is not None and \
+           self.appmenu_applet_available and \
+           self.brisk_menu_available and \
+           not self.indicators_available:
+            if self.panel_layout_exists('eleven-no-indicators'):
+                panels.append([_("Cupertino (no indicators)"), "eleven-no-indicators"])
+
         self.builder.get_object("combobox_panels").set_model(panels)
 
     def ask_for_layout_name(self):
