diff -ru --no-dereference a/files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py b/files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py
--- a/files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py	2017-05-04 17:03:58.000000000 +0200
+++ b/files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py	2017-05-14 01:10:48.668720728 +0200
@@ -85,8 +85,11 @@
     [_("Software Sources"),              "mintsources",                  "cs-sources",         "admin",          _("ppa, repository, package, source, download")],
     [_("Users and Groups"),              "cinnamon-settings-users",      "cs-user-accounts",   "admin",          _("user, users, account, accounts, group, groups, password")],
     [_("Bluetooth"),                     "blueberry",                    "cs-bluetooth",       "hardware",       _("bluetooth, dongle, transfer, mobile")],
-    [_("Manage Services and Units"),     "systemd-manager-pkexec",       "cs-sources",         "admin",          _("systemd, units, services, systemctl, init")]
-]
+    [_("Manage Services and Units"),     "systemd-manager-pkexec",       "cs-sources",         "admin",          _("systemd, units, services, systemctl, init")],
+    [_("Manjaro Settings Manager"),      "manjaro-settings-manager",     "cs-cat-hardware",    "admin",          _("manjaro, settings, manager, kernel, graphics, driver, foreign, language, user, account, time, date, keyboard, hardware")],
+    [_("Manjaro Notifier Settings"),     "msm_notifier --settings",      "cs-sources",         "admin",          _("manjaro, settings, notifier, kernel, language, foreign")],
+    [_("About Me"),                      "mugshot",                      "cs-user",            "admin",          _("user, account, avatar")]
+ ]
 
 def print_timing(func):
     # decorate functions with @print_timing to output how long they take to run.
