diff -Naur a/data/lightdm-gtk-greeter.conf b/data/lightdm-gtk-greeter.conf
--- a/data/lightdm-gtk-greeter.conf	2014-10-20 18:16:01.000000000 +1100
+++ b/data/lightdm-gtk-greeter.conf	2014-10-27 13:02:15.005501000 +1100
@@ -19,6 +19,7 @@
 # default-user-image = Image used as default user icon, path or #icon-name
 # hide-user-image = true|false, false by default
 # screensaver-timeout = Timeout (in seconds) until the screen blanks when the greeter is called as lockscreen
+# panel-position = can be set to 'bottom'
 #
 # Template for monitor configuration:
 # [monitor: name]
@@ -43,3 +44,4 @@
 #reader=
 #position=
 #screensaver-timeout=
+#panel-position=
diff -Naur a/src/lightdm-gtk-greeter.c b/src/lightdm-gtk-greeter.c
--- a/src/lightdm-gtk-greeter.c	2014-10-20 18:16:01.000000000 +1100
+++ b/src/lightdm-gtk-greeter.c	2014-10-27 12:59:32.589276000 +1100
@@ -108,10 +108,11 @@
 
 /* Function translate user defined coordinates to absolute value */
 static gint get_absolute_position (const DimensionPosition *p, gint screen, gint window);
-static const WindowPosition WINDOW_POS_CENTER   = {.x = { 50, +1, TRUE,   0}, .y = { 50, +1, TRUE,   0}};
-static const WindowPosition WINDOW_POS_TOP_LEFT = {.x = {  0, +1, FALSE, -1}, .y = {  0, +1, FALSE, -1}};
-static const WindowPosition ONBOARD_WINDOW_POS  = {.x = { 50, +1, TRUE,   0}, .y = {  0, -1, FALSE, +1}};
-static const WindowPosition ONBOARD_WINDOW_SIZE = {.x = {610,  0, FALSE,  0}, .y = {210,  0, FALSE,  0}};
+static const WindowPosition WINDOW_POS_CENTER      = {.x = { 50, +1, TRUE,   0}, .y = { 50, +1, TRUE,   0}};
+static const WindowPosition WINDOW_POS_TOP_LEFT    = {.x = {  0, +1, FALSE, -1}, .y = {  0, +1, FALSE, -1}};
+static const WindowPosition WINDOW_POS_BOTTOM_LEFT = {.x = {  0, +1, FALSE, -1}, .y = {100, +1, TRUE,  -1}};
+static const WindowPosition ONBOARD_WINDOW_POS     = {.x = { 50, +1, TRUE,   0}, .y = {  0, -1, FALSE, +1}};
+static const WindowPosition ONBOARD_WINDOW_SIZE    = {.x = {610,  0, FALSE,  0}, .y = {210,  0, FALSE,  0}};
 static WindowPosition main_window_pos;
 static WindowPosition panel_window_pos;
 
@@ -2942,7 +2943,13 @@
 
         g_free (value);
     }
-    panel_window_pos = WINDOW_POS_TOP_LEFT;
+
+    value = g_key_file_get_value (config, "greeter", "panel-position", NULL);
+    if (g_strcmp0 (value, "bottom") == 0)
+        panel_window_pos = WINDOW_POS_BOTTOM_LEFT;
+    else
+        panel_window_pos = WINDOW_POS_TOP_LEFT;
+    g_free (value);
 
     gtk_builder_connect_signals(builder, greeter);
 
