diff -rupN a/modules/keyboard/keyboardwidget.cpp b/modules/keyboard/keyboardwidget.cpp
--- a/modules/keyboard/keyboardwidget.cpp	2017-08-18 11:02:31.000000000 +0200
+++ b/modules/keyboard/keyboardwidget.cpp	2017-09-04 00:10:01.413031700 +0200
@@ -89,7 +89,7 @@ KeyboardWidget::KeyboardWidget(KeyboardM
 
     SettingsGroup* scGroup = new SettingsGroup();
     m_scItem = new NextPageWidget();
-    m_scItem->setTitle(tr("Shortcut"));
+    m_scItem->setTitle(tr("Keyboard Shortcuts"));
     scGroup->appendItem(m_scItem);
 
     m_centralLayout->addWidget(keyTest);
diff -rupN a/translations/dde-control-center_af.ts b/translations/dde-control-center_af.ts
--- a/translations/dde-control-center_af.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_af.ts	2017-09-04 00:18:15.832146245 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Kanselleer</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_am_ET.ts b/translations/dde-control-center_am_ET.ts
--- a/translations/dde-control-center_am_ET.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_am_ET.ts	2017-09-04 00:21:05.444035451 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>አቋራጭ </translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>መሰረዣ</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ar.ts b/translations/dde-control-center_ar.ts
--- a/translations/dde-control-center_ar.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ar.ts	2017-09-04 00:20:54.600794467 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>إختصار</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>إلغاء</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ast.ts b/translations/dde-control-center_ast.ts
--- a/translations/dde-control-center_ast.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ast.ts	2017-09-04 00:20:42.670896068 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Atayu</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Encaboxar</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_az.ts b/translations/dde-control-center_az.ts
--- a/translations/dde-control-center_az.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_az.ts	2017-09-04 00:20:32.830979866 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>İmtina</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_bg.ts b/translations/dde-control-center_bg.ts
--- a/translations/dde-control-center_bg.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_bg.ts	2017-09-04 00:20:20.717749686 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation> Кратък път</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Отказ</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_bn.ts b/translations/dde-control-center_bn.ts
--- a/translations/dde-control-center_bn.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_bn.ts	2017-09-04 00:21:36.067107953 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>বাতিল</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ca.ts b/translations/dde-control-center_ca.ts
--- a/translations/dde-control-center_ca.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ca.ts	2017-09-04 00:21:53.066963145 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancel·la</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_cs.ts b/translations/dde-control-center_cs.ts
--- a/translations/dde-control-center_cs.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_cs.ts	2017-09-04 00:22:10.786812193 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Klávesová zkratka</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Zrušit</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_da.ts b/translations/dde-control-center_da.ts
--- a/translations/dde-control-center_da.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_da.ts	2017-09-04 00:22:22.900042332 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Genvej</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Annuller</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_de.ts b/translations/dde-control-center_de.ts
--- a/translations/dde-control-center_de.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_de.ts	2017-09-04 00:22:35.953264457 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Tastenkürzel</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Abbrechen</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_el.ts b/translations/dde-control-center_el.ts
--- a/translations/dde-control-center_el.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_el.ts	2017-09-04 00:22:48.926487258 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Ακύρωση</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_en_AU.ts b/translations/dde-control-center_en_AU.ts
--- a/translations/dde-control-center_en_AU.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_en_AU.ts	2017-09-04 00:23:04.413021973 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancel</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_eo.ts b/translations/dde-control-center_eo.ts
--- a/translations/dde-control-center_eo.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_eo.ts	2017-09-04 00:23:17.876240588 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Nuligi</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_es_419.ts b/translations/dde-control-center_es_419.ts
--- a/translations/dde-control-center_es_419.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_es_419.ts	2017-09-04 00:23:43.702687176 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Acceso Directo</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancelar</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_es.ts b/translations/dde-control-center_es.ts
--- a/translations/dde-control-center_es.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_es.ts	2017-09-04 00:23:31.079461414 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Atajo de teclado</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancelar</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_fa.ts b/translations/dde-control-center_fa.ts
--- a/translations/dde-control-center_fa.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fa.ts	2017-09-04 00:23:57.232571874 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>انصراف</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_fil.ts b/translations/dde-control-center_fil.ts
--- a/translations/dde-control-center_fil.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fil.ts	2017-09-04 00:24:26.358990305 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation type="unfinished"/>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_fi.ts b/translations/dde-control-center_fi.ts
--- a/translations/dde-control-center_fi.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fi.ts	2017-09-04 00:24:12.272443697 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Peruuta</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_fr.ts b/translations/dde-control-center_fr.ts
--- a/translations/dde-control-center_fr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fr.ts	2017-09-04 00:24:39.192214256 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Raccourci </translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Annuler</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_gl_ES.ts b/translations/dde-control-center_gl_ES.ts
--- a/translations/dde-control-center_gl_ES.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_gl_ES.ts	2017-09-04 00:24:50.375452268 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Atallo</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancelar</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_he.ts b/translations/dde-control-center_he.ts
--- a/translations/dde-control-center_he.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_he.ts	2017-09-04 00:25:04.551998092 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>ביטול</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_hi_IN.ts b/translations/dde-control-center_hi_IN.ts
--- a/translations/dde-control-center_hi_IN.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hi_IN.ts	2017-09-04 00:25:18.255214613 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>रद्द करो</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_hr.ts b/translations/dde-control-center_hr.ts
--- a/translations/dde-control-center_hr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hr.ts	2017-09-04 00:25:39.011704332 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Prečac</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Otkaži</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_hu.ts b/translations/dde-control-center_hu.ts
--- a/translations/dde-control-center_hu.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hu.ts	2017-09-04 00:25:52.938252268 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Gyorsbillentyű</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Mégse</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_hy.ts b/translations/dde-control-center_hy.ts
--- a/translations/dde-control-center_hy.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hy.ts	2017-09-04 00:26:05.284813670 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Չեղարկել</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_id.ts b/translations/dde-control-center_id.ts
--- a/translations/dde-control-center_id.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_id.ts	2017-09-04 00:26:20.028021299 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Pintasan</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Batalkan</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_it.ts b/translations/dde-control-center_it.ts
--- a/translations/dde-control-center_it.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_it.ts	2017-09-04 00:26:38.451197543 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Scorciatoia</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancella</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ja.ts b/translations/dde-control-center_ja.ts
--- a/translations/dde-control-center_ja.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ja.ts	2017-09-04 00:27:03.100987345 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>キャンセル</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ko.ts b/translations/dde-control-center_ko.ts
--- a/translations/dde-control-center_ko.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ko.ts	2017-09-04 00:27:18.530855761 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>단축키</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>취소</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ku_IQ.ts b/translations/dde-control-center_ku_IQ.ts
--- a/translations/dde-control-center_ku_IQ.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ku_IQ.ts	2017-09-04 00:27:33.687393166 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>پاشگەزبوونەوە</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_lt.ts b/translations/dde-control-center_lt.ts
--- a/translations/dde-control-center_lt.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_lt.ts	2017-09-04 00:27:48.370601270 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Trumpinys</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Atsisakyti</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ml.ts b/translations/dde-control-center_ml.ts
--- a/translations/dde-control-center_ml.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ml.ts	2017-09-04 00:28:05.410455933 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>റദ്ദാക്കുക</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_mn.ts b/translations/dde-control-center_mn.ts
--- a/translations/dde-control-center_mn.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_mn.ts	2017-09-04 00:28:19.267003786 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Цуцлах</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ms.ts b/translations/dde-control-center_ms.ts
--- a/translations/dde-control-center_ms.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ms.ts	2017-09-04 00:28:31.096900628 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Pintasan</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Batal</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_nb.ts b/translations/dde-control-center_nb.ts
--- a/translations/dde-control-center_nb.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_nb.ts	2017-09-04 00:28:45.640107145 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Avbryt</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_nl.ts b/translations/dde-control-center_nl.ts
--- a/translations/dde-control-center_nl.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_nl.ts	2017-09-04 00:28:56.376680192 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Sneltoets</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Annuleren</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_pam.ts b/translations/dde-control-center_pam.ts
--- a/translations/dde-control-center_pam.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pam.ts	2017-09-04 00:29:08.849904765 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>I-cancel</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_pl.ts b/translations/dde-control-center_pl.ts
--- a/translations/dde-control-center_pl.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pl.ts	2017-09-04 00:29:21.269796472 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Skrót</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Anuluj</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_pt_BR.ts b/translations/dde-control-center_pt_BR.ts
--- a/translations/dde-control-center_pt_BR.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pt_BR.ts	2017-09-04 00:29:45.779582773 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Atalho</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancelar</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_pt.ts b/translations/dde-control-center_pt.ts
--- a/translations/dde-control-center_pt.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pt.ts	2017-09-04 00:29:33.019694025 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Atalhos</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Cancelar</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ro.ts b/translations/dde-control-center_ro.ts
--- a/translations/dde-control-center_ro.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ro.ts	2017-09-04 00:30:00.746118953 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Anulează</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ru.ts b/translations/dde-control-center_ru.ts
--- a/translations/dde-control-center_ru.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ru.ts	2017-09-04 00:30:13.336009190 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Сочетание Клавиш </translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Отмена</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_sk.ts b/translations/dde-control-center_sk.ts
--- a/translations/dde-control-center_sk.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sk.ts	2017-09-04 00:30:28.399211201 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Odkaz</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Zrušiť</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_sl.ts b/translations/dde-control-center_sl.ts
--- a/translations/dde-control-center_sl.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sl.ts	2017-09-04 00:30:41.352431610 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Bližnjica</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Prekliči</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_sr.ts b/translations/dde-control-center_sr.ts
--- a/translations/dde-control-center_sr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sr.ts	2017-09-04 00:30:53.492325778 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Пречица</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Откажи</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_sv.ts b/translations/dde-control-center_sv.ts
--- a/translations/dde-control-center_sv.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sv.ts	2017-09-04 00:31:17.338784568 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Genväg</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Avbryt</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_ta.ts b/translations/dde-control-center_ta.ts
--- a/translations/dde-control-center_ta.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ta.ts	2017-09-04 00:31:28.755351716 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>நிறுத்து</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_tr.ts b/translations/dde-control-center_tr.ts
--- a/translations/dde-control-center_tr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_tr.ts	2017-09-04 00:31:41.255242757 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Kısayol</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>İptal</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center.ts b/translations/dde-control-center.ts
--- a/translations/dde-control-center.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center.ts	2017-09-04 00:15:36.116838588 +0200
@@ -1053,7 +1053,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
diff -rupN a/translations/dde-control-center_ug.ts b/translations/dde-control-center_ug.ts
--- a/translations/dde-control-center_ug.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ug.ts	2017-09-04 00:31:55.595117762 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>تىزلەتمە كونۇپكا</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>بىكار قىلىش</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_uk.ts b/translations/dde-control-center_uk.ts
--- a/translations/dde-control-center_uk.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_uk.ts	2017-09-04 00:32:06.738353967 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Ярлик</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Відмінити</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_vi.ts b/translations/dde-control-center_vi.ts
--- a/translations/dde-control-center_vi.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_vi.ts	2017-09-04 00:32:18.684916505 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>Phím tắt</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>Hủy</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_zh_CN.ts b/translations/dde-control-center_zh_CN.ts
--- a/translations/dde-control-center_zh_CN.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_zh_CN.ts	2017-09-04 00:32:32.334797534 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>快捷键</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>取消</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_zh_HK.ts b/translations/dde-control-center_zh_HK.ts
--- a/translations/dde-control-center_zh_HK.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_zh_HK.ts	2017-09-04 00:32:44.118028167 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation type="unfinished"/>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>取消</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
diff -rupN a/translations/dde-control-center_zh_TW.ts b/translations/dde-control-center_zh_TW.ts
--- a/translations/dde-control-center_zh_TW.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_zh_TW.ts	2017-09-04 00:32:57.254580342 +0200
@@ -1048,7 +1048,7 @@
     </message>
     <message>
         <location filename="../modules/keyboard/keyboardwidget.cpp" line="92"/>
-        <source>Shortcut</source>
+        <source>Keyboard Shortcuts</source>
         <translation>快速鍵</translation>
     </message>
 </context>
@@ -2215,4 +2215,4 @@
         <translation>完成</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
