diff -rupN a/modules/systeminfo/systeminfomodel.cpp b/modules/systeminfo/systeminfomodel.cpp
--- a/modules/systeminfo/systeminfomodel.cpp	2017-08-18 11:02:31.000000000 +0200
+++ b/modules/systeminfo/systeminfomodel.cpp	2017-09-04 00:36:39.529310204 +0200
@@ -93,15 +93,6 @@ void SystemInfoModel::setDistroVer(const
     emit distroverChanged(m_distrover);
 }
 
-void SystemInfoModel::setVersion(const QString &version)
-{
-    if(m_version == version)
-        return;
-
-    m_version = version;
-    emit versionChanged(m_version);
-}
-
 void SystemInfoModel::setType(qlonglong type)
 {
     if(m_type == QString("%1").arg(type))
diff -rupN a/modules/systeminfo/systeminfomodel.h b/modules/systeminfo/systeminfomodel.h
--- a/modules/systeminfo/systeminfomodel.h	2017-08-18 11:02:31.000000000 +0200
+++ b/modules/systeminfo/systeminfomodel.h	2017-09-04 00:36:39.529310204 +0200
@@ -20,7 +20,6 @@ public:
 
     QString distroid() const { return m_distroid;}
     QString distrover() const { return m_distrover;}
-    QString version() const { return m_version;}
     QString type() const { return m_type;}
     QString processor() const { return m_processor;}
     QString memory() const { return m_memory;}
@@ -57,7 +56,6 @@ public slots:
 #endif
     void setDistroID(const QString& distroid);
     void setDistroVer(const QString& distrover);
-    void setVersion(const QString& version);
     void setType(qlonglong type);
     void setProcessor(const QString& processor);
     void setMemory(qulonglong memory);
@@ -74,7 +72,6 @@ private:
 
     QString m_distroid;
     QString m_distrover;
-    QString m_version;
     QString m_type;
     QString m_processor;
     QString m_memory;
diff -rupN a/modules/systeminfo/systeminfowidget.cpp b/modules/systeminfo/systeminfowidget.cpp
--- a/modules/systeminfo/systeminfowidget.cpp	2017-08-18 11:02:31.000000000 +0200
+++ b/modules/systeminfo/systeminfowidget.cpp	2017-09-04 00:43:20.285820158 +0200
@@ -58,16 +58,12 @@ SystemInfoWidget::SystemInfoWidget(Syste
     m_distroid->setVisible(false);
 
     m_distrover = new TitleValueItem();
-    m_distrover->setTitle(tr("Distribution Release:"));
+    m_distrover->setTitle(tr("Release:"));
     m_distrover->setValue(m_model->distrover());
     m_distrover->setVisible(false);
 
-    m_version = new TitleValueItem();
-    m_version->setTitle(tr("Edition License:"));
-    m_version->setValue(m_model->version());
-
     m_type = new TitleValueItem();
-    m_type->setTitle(tr("Type:"));
+    m_type->setTitle(tr("Architecture:"));
 
     m_processor = new TitleValueItem();
     m_processor->setTitle(tr("Processor:"));
@@ -84,7 +80,6 @@ SystemInfoWidget::SystemInfoWidget(Syste
     infoGroup->appendItem(logo);
     infoGroup->appendItem(m_distroid);
     infoGroup->appendItem(m_distrover);
-    infoGroup->appendItem(m_version);
     infoGroup->appendItem(m_type);
     infoGroup->appendItem(m_processor);
     infoGroup->appendItem(m_memory);
@@ -112,7 +107,6 @@ SystemInfoWidget::SystemInfoWidget(Syste
 #endif
     connect(m_model, SIGNAL(distroidChanged(QString)), this, SLOT(setDistroID(QString)));
     connect(m_model, SIGNAL(distroverChanged(QString)), this, SLOT(setDistroVer(QString)));
-    connect(m_model, SIGNAL(versionChanged(QString)), this, SLOT(setEdition(QString)));
     connect(m_model, SIGNAL(typeChanged(QString)), this, SLOT(setType(QString)));
     connect(m_model, SIGNAL(processorChanged(QString)), this, SLOT(setProcessor(QString)));
     connect(m_model, SIGNAL(memoryChanged(QString)), this, SLOT(setMemory(QString)));
@@ -140,11 +134,6 @@ void SystemInfoWidget::setDistroVer(cons
     m_distrover->setValue(distrover);
 }
 
-void SystemInfoWidget::setEdition(const QString &edition)
-{
-    m_version->setValue(edition);
-}
-
 void SystemInfoWidget::setType(const QString &type)
 {
     m_type->setValue(tr("%1Bit").arg(type));
diff -rupN a/modules/systeminfo/systeminfowidget.h b/modules/systeminfo/systeminfowidget.h
--- a/modules/systeminfo/systeminfowidget.h	2017-08-18 11:02:31.000000000 +0200
+++ b/modules/systeminfo/systeminfowidget.h	2017-09-04 00:36:39.529310204 +0200
@@ -31,7 +31,6 @@ signals:
     void boot();
 
 public slots:
-    void setEdition(const QString& edition);
     void setType(const QString& type);
     void setProcessor(const QString& processor);
     void setMemory(const QString& memory);
@@ -42,7 +41,6 @@ public slots:
 private:
     widgets::TitleValueItem* m_distroid;
     widgets::TitleValueItem* m_distrover;
-    widgets::TitleValueItem* m_version;
     widgets::TitleValueItem* m_type;
     widgets::TitleValueItem* m_processor;
     widgets::TitleValueItem* m_memory;
diff -rupN a/modules/systeminfo/systeminfowork.cpp b/modules/systeminfo/systeminfowork.cpp
--- a/modules/systeminfo/systeminfowork.cpp	2017-08-18 11:02:31.000000000 +0200
+++ b/modules/systeminfo/systeminfowork.cpp	2017-09-04 00:36:39.529310204 +0200
@@ -30,7 +30,6 @@ SystemInfoWork::SystemInfoWork(SystemInf
 
     connect(m_systemInfoInter, &__SystemInfo::DistroIDChanged, m_model, &SystemInfoModel::setDistroID);
     connect(m_systemInfoInter, &__SystemInfo::DistroVerChanged, m_model, &SystemInfoModel::setDistroVer);
-    connect(m_systemInfoInter, &__SystemInfo::VersionChanged, m_model, &SystemInfoModel::setVersion);
     connect(m_systemInfoInter, &__SystemInfo::SystemTypeChanged, m_model, &SystemInfoModel::setType);
     connect(m_systemInfoInter, &__SystemInfo::ProcessorChanged, m_model, &SystemInfoModel::setProcessor);
     connect(m_systemInfoInter, &__SystemInfo::MemoryCapChanged, m_model, &SystemInfoModel::setMemory);
@@ -41,7 +40,6 @@ void SystemInfoWork::activate()
 {
     m_model->setDistroID(m_systemInfoInter->distroID());
     m_model->setDistroVer(m_systemInfoInter->distroVer());
-    m_model->setVersion(m_systemInfoInter->version());
     m_model->setType(m_systemInfoInter->systemType());
     m_model->setProcessor(m_systemInfoInter->processor());
     m_model->setMemory(m_systemInfoInter->memoryCap());
diff -rupN a/translations/dde-control-center_af.ts b/translations/dde-control-center_af.ts
--- a/translations/dde-control-center_af.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_af.ts	2017-09-04 00:44:31.185202968 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipe:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_am_ET.ts b/translations/dde-control-center_am_ET.ts
--- a/translations/dde-control-center_am_ET.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_am_ET.ts	2017-09-04 00:44:31.411867662 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>የ ተለቀቀው ስርጭት:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>አይነት:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ar.ts b/translations/dde-control-center_ar.ts
--- a/translations/dde-control-center_ar.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ar.ts	2017-09-04 00:44:31.388534532 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>إصدار التوزيعة:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>النوع</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ast.ts b/translations/dde-control-center_ast.ts
--- a/translations/dde-control-center_ast.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ast.ts	2017-09-04 00:44:31.341868272 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Llanzamientu de distribución:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Triba:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_az.ts b/translations/dde-control-center_az.ts
--- a/translations/dde-control-center_az.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_az.ts	2017-09-04 00:44:31.321868446 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Növ:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_bg.ts b/translations/dde-control-center_bg.ts
--- a/translations/dde-control-center_bg.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_bg.ts	2017-09-04 00:44:31.155203230 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Излизане на дистрибуцията:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Тип:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_bn.ts b/translations/dde-control-center_bn.ts
--- a/translations/dde-control-center_bn.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_bn.ts	2017-09-04 00:44:31.151869925 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>ধরণ:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ca.ts b/translations/dde-control-center_ca.ts
--- a/translations/dde-control-center_ca.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ca.ts	2017-09-04 00:44:31.318535141 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipus:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_cs.ts b/translations/dde-control-center_cs.ts
--- a/translations/dde-control-center_cs.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_cs.ts	2017-09-04 00:44:31.301868620 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Vydání distribuce:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Typ:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_da.ts b/translations/dde-control-center_da.ts
--- a/translations/dde-control-center_da.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_da.ts	2017-09-04 00:44:31.381867924 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distributionsudgivelse:</translation>
     </message>
     <message>
@@ -1909,8 +1909,8 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
-        <translation>Type:</translation>
+        <source>Architecture:</source>
+        <translation>Architecture:</translation>
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="98"/>
diff -rupN a/translations/dde-control-center_de.ts b/translations/dde-control-center_de.ts
--- a/translations/dde-control-center_de.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_de.ts	2017-09-04 00:44:31.368534706 +0200
@@ -1899,8 +1899,8 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
-        <translation>Distribution Release:</translation>
+        <source>Release:</source>
+        <translation>Release:</translation>
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="66"/>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Typ:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_el.ts b/translations/dde-control-center_el.ts
--- a/translations/dde-control-center_el.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_el.ts	2017-09-04 00:44:31.398534445 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Τύπος:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_en_AU.ts b/translations/dde-control-center_en_AU.ts
--- a/translations/dde-control-center_en_AU.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_en_AU.ts	2017-09-04 00:44:31.228535925 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,8 +1909,8 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
-        <translation>Type:</translation>
+        <source>Architecture:</source>
+        <translation>Architecture:</translation>
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="98"/>
diff -rupN a/translations/dde-control-center_eo.ts b/translations/dde-control-center_eo.ts
--- a/translations/dde-control-center_eo.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_eo.ts	2017-09-04 00:44:31.361868097 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_es_419.ts b/translations/dde-control-center_es_419.ts
--- a/translations/dde-control-center_es_419.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_es_419.ts	2017-09-04 00:44:31.145203317 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distribución lanzada:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_es.ts b/translations/dde-control-center_es.ts
--- a/translations/dde-control-center_es.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_es.ts	2017-09-04 00:44:31.415200967 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Versión de la distribución</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_fa.ts b/translations/dde-control-center_fa.ts
--- a/translations/dde-control-center_fa.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fa.ts	2017-09-04 00:44:31.358534793 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>نوع:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_fil.ts b/translations/dde-control-center_fil.ts
--- a/translations/dde-control-center_fil.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fil.ts	2017-09-04 00:44:31.328535054 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_fi.ts b/translations/dde-control-center_fi.ts
--- a/translations/dde-control-center_fi.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fi.ts	2017-09-04 00:44:31.308535229 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tyyppi:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_fr.ts b/translations/dde-control-center_fr.ts
--- a/translations/dde-control-center_fr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_fr.ts	2017-09-04 00:44:31.211869403 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distribution :</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Type : </translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_gl_ES.ts b/translations/dde-control-center_gl_ES.ts
--- a/translations/dde-control-center_gl_ES.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_gl_ES.ts	2017-09-04 00:44:31.285202098 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distribución publicada:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_he.ts b/translations/dde-control-center_he.ts
--- a/translations/dde-control-center_he.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_he.ts	2017-09-04 00:44:31.261868968 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>סוג</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_hi_IN.ts b/translations/dde-control-center_hi_IN.ts
--- a/translations/dde-control-center_hi_IN.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hi_IN.ts	2017-09-04 00:44:31.271868881 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>प्रकार:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_hr.ts b/translations/dde-control-center_hr.ts
--- a/translations/dde-control-center_hr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hr.ts	2017-09-04 00:44:31.201869490 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Izdanje distribucije:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Vrsta:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_hu.ts b/translations/dde-control-center_hu.ts
--- a/translations/dde-control-center_hu.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hu.ts	2017-09-04 00:44:31.268535576 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Terjesztés kiadása:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Típus:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_hy.ts b/translations/dde-control-center_hy.ts
--- a/translations/dde-control-center_hy.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_hy.ts	2017-09-04 00:44:31.455200618 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_id.ts b/translations/dde-control-center_id.ts
--- a/translations/dde-control-center_id.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_id.ts	2017-09-04 00:44:31.351868184 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Rilis Distribusi:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipe:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_it.ts b/translations/dde-control-center_it.ts
--- a/translations/dde-control-center_it.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_it.ts	2017-09-04 00:44:31.405201054 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Release distribuzione: </translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ja.ts b/translations/dde-control-center_ja.ts
--- a/translations/dde-control-center_ja.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ja.ts	2017-09-04 00:44:31.345201576 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>種類:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ko.ts b/translations/dde-control-center_ko.ts
--- a/translations/dde-control-center_ko.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ko.ts	2017-09-04 00:44:31.198536186 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>종류:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ku_IQ.ts b/translations/dde-control-center_ku_IQ.ts
--- a/translations/dde-control-center_ku_IQ.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ku_IQ.ts	2017-09-04 00:44:31.288535403 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_lt.ts b/translations/dde-control-center_lt.ts
--- a/translations/dde-control-center_lt.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_lt.ts	2017-09-04 00:44:31.135203404 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distribucijos laida:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipas:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ml.ts b/translations/dde-control-center_ml.ts
--- a/translations/dde-control-center_ml.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ml.ts	2017-09-04 00:44:31.165203143 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_mn.ts b/translations/dde-control-center_mn.ts
--- a/translations/dde-control-center_mn.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_mn.ts	2017-09-04 00:44:31.375201315 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ms.ts b/translations/dde-control-center_ms.ts
--- a/translations/dde-control-center_ms.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ms.ts	2017-09-04 00:44:31.461867227 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Keluaran Distribusi:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Jenis:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_nb.ts b/translations/dde-control-center_nb.ts
--- a/translations/dde-control-center_nb.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_nb.ts	2017-09-04 00:44:31.255202359 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,8 +1909,8 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
-        <translation>Type:</translation>
+        <source>Architecture:</source>
+        <translation>Architecture:</translation>
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="98"/>
diff -rupN a/translations/dde-control-center_nl.ts b/translations/dde-control-center_nl.ts
--- a/translations/dde-control-center_nl.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_nl.ts	2017-09-04 00:44:31.218536012 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distributie Release:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Soort:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_pam.ts b/translations/dde-control-center_pam.ts
--- a/translations/dde-control-center_pam.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pam.ts	2017-09-04 00:44:31.158536534 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_pl.ts b/translations/dde-control-center_pl.ts
--- a/translations/dde-control-center_pl.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pl.ts	2017-09-04 00:44:31.221869317 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Wydanie dystrybucji:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Rodzaj:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_pt_BR.ts b/translations/dde-control-center_pt_BR.ts
--- a/translations/dde-control-center_pt_BR.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pt_BR.ts	2017-09-04 00:44:31.355201489 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distribuição Release:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_pt.ts b/translations/dde-control-center_pt.ts
--- a/translations/dde-control-center_pt.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_pt.ts	2017-09-04 00:44:31.208536099 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Data de Lançamento da Distribuição:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tipo:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ro.ts b/translations/dde-control-center_ro.ts
--- a/translations/dde-control-center_ro.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ro.ts	2017-09-04 00:44:31.235202533 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tip:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ru.ts b/translations/dde-control-center_ru.ts
--- a/translations/dde-control-center_ru.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ru.ts	2017-09-04 00:44:31.171869752 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Распространяемый Выпуск:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Тип:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_sk.ts b/translations/dde-control-center_sk.ts
--- a/translations/dde-control-center_sk.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sk.ts	2017-09-04 00:44:31.331868359 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Vydanie distribúcie:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Typ:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_sl.ts b/translations/dde-control-center_sl.ts
--- a/translations/dde-control-center_sl.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sl.ts	2017-09-04 00:44:31.401867749 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Distribucija:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Vrsta:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_sr.ts b/translations/dde-control-center_sr.ts
--- a/translations/dde-control-center_sr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sr.ts	2017-09-04 00:44:31.425200880 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Врста:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_sv.ts b/translations/dde-control-center_sv.ts
--- a/translations/dde-control-center_sv.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_sv.ts	2017-09-04 00:44:31.305201924 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Typ:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ta.ts b/translations/dde-control-center_ta.ts
--- a/translations/dde-control-center_ta.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ta.ts	2017-09-04 00:44:31.295202011 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_tr.ts b/translations/dde-control-center_tr.ts
--- a/translations/dde-control-center_tr.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_tr.ts	2017-09-04 00:44:31.315201837 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Dağıtım Yayını:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Tür:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center.ts b/translations/dde-control-center.ts
--- a/translations/dde-control-center.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center.ts	2017-09-04 00:44:31.441867401 +0200
@@ -1904,7 +1904,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -1914,7 +1914,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_ug.ts b/translations/dde-control-center_ug.ts
--- a/translations/dde-control-center_ug.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_ug.ts	2017-09-04 00:44:31.245202446 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>تارقىتىشى: </translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>تىپى: </translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_uk.ts b/translations/dde-control-center_uk.ts
--- a/translations/dde-control-center_uk.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_uk.ts	2017-09-04 00:44:31.431867489 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Розповсюдження Релізу: </translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Тип:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_vi.ts b/translations/dde-control-center_vi.ts
--- a/translations/dde-control-center_vi.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_vi.ts	2017-09-04 00:44:31.178536360 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>Phát hành Phân phối:</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>Loại:</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_zh_CN.ts b/translations/dde-control-center_zh_CN.ts
--- a/translations/dde-control-center_zh_CN.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_zh_CN.ts	2017-09-04 00:44:31.338534967 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>发行版</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>类型：</translation>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_zh_HK.ts b/translations/dde-control-center_zh_HK.ts
--- a/translations/dde-control-center_zh_HK.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_zh_HK.ts	2017-09-04 00:44:31.445200705 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation type="unfinished"/>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation type="unfinished"/>
     </message>
     <message>
diff -rupN a/translations/dde-control-center_zh_TW.ts b/translations/dde-control-center_zh_TW.ts
--- a/translations/dde-control-center_zh_TW.ts	2017-08-18 11:02:31.000000000 +0200
+++ b/translations/dde-control-center_zh_TW.ts	2017-09-04 00:44:31.278535489 +0200
@@ -1899,7 +1899,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="61"/>
-        <source>Distribution Release:</source>
+        <source>Release:</source>
         <translation>發行方式：</translation>
     </message>
     <message>
@@ -1909,7 +1909,7 @@
     </message>
     <message>
         <location filename="../modules/systeminfo/systeminfowidget.cpp" line="70"/>
-        <source>Type:</source>
+        <source>Architecture:</source>
         <translation>類型：</translation>
     </message>
     <message>
