diff -rupN a/plugins/navgation/navwidget.cpp b/plugins/navgation/navwidget.cpp
--- a/plugins/navgation/navwidget.cpp	2017-07-14 07:42:31.000000000 +0200
+++ b/plugins/navgation/navwidget.cpp	2017-07-19 01:39:26.108689605 +0200
@@ -3,7 +3,7 @@
 
 #include <QTimer>
 
-const QStringList MODULES = { "accounts", "display", "defapp", "personalization", "network", "bluetooth", "sound", "datetime", "power", "mouse", "keyboard", "wacom", "update", "systeminfo" };
+const QStringList MODULES = { "accounts", "display", "defapp", "personalization", "network", "bluetooth", "sound", "datetime", "power", "mouse", "keyboard", "wacom", "systeminfo" };
 
 NavWidget::NavWidget(QWidget *parent)
     : QWidget(parent),
@@ -104,7 +104,6 @@ void NavWidget::setTipsText(const QStrin
         QT_TRANSLATE_NOOP("dcc::mouse::MouseWidget", "Mouse"),
         QT_TRANSLATE_NOOP("dcc::keyboard::KeyboardWidget", "Keyboard and Language"),
         QT_TRANSLATE_NOOP("dcc::wacom::WacomWidget", "Wacom"),
-        QT_TRANSLATE_NOOP("dcc::update::UpdateCtrlWidget", "Update"),
         QT_TRANSLATE_NOOP("dcc::systeminfo::SystemInfoWidget", "System Information"),
     };
 
@@ -121,7 +120,6 @@ void NavWidget::setTipsText(const QStrin
         "dcc::mouse::MouseWidget",
         "dcc::keyboard::KeyboardWidget",
         "dcc::wacom::WacomWidget",
-        "dcc::update::UpdateCtrlWidget",
         "dcc::systeminfo::SystemInfoWidget",
     };
 
