diff -rupN src/slick-greeter.vala src-mod/slick-greeter.vala
--- src/slick-greeter.vala	2017-05-05 15:40:53.000000000 +0200
+++ src-mod/slick-greeter.vala	2017-05-08 18:40:05.185315821 +0200
@@ -184,7 +184,7 @@ public class SlickGreeter
     {
         /* Make sure the given session actually exists. Return it if it does.
         otherwise, return the default session. */
-        var path = Path.build_filename  ("/usr/share/xsessions/", session, null);
+        var path = Path.build_filename ("/usr/share/xsessions/", session.concat(".desktop"), null);
         if (session != null && FileUtils.test (path, FileTest.EXISTS) ) {
             return session;
         }
