--- slick-greeter.vala	2017-06-29 19:36:29.964013818 +0200
+++ slick-greeter.vala.new	2017-06-29 23:03:15.780024760 +0200
@@ -183,6 +183,7 @@ public class SlickGreeter
         sessions.append ("plasma");
         sessions.append ("kde-plasma");
         sessions.append ("kde");
+        sessions.append ("budgie-desktop");
         sessions.append ("gnome");
 
         foreach (string session in sessions) {
