--- extension.js	2016-11-14 11:55:53.000000000 +0100
+++ extension-new.js	2016-11-18 22:22:58.129127183 +0100
@@ -771,7 +771,7 @@ TaskBar.prototype =
         if ((this.settings.get_boolean("first-start")) && (Main.sessionMode.currentMode === 'user'))
         {
             //Comment out the next line to disable the preferences window from opening at the first start
-            Main.Util.trySpawnCommandLine('gnome-shell-extension-prefs ' + Extension.metadata.uuid);
+            //Main.Util.trySpawnCommandLine('gnome-shell-extension-prefs ' + Extension.metadata.uuid);
             this.settings.set_boolean("first-start", false);
         }
         //Find out if the bottom panel extension is enabled
