--- a/src/r8168_n.c	2016-06-04 16:12:05.746880000 +0200
+++ b/src/r8168_n.c	2016-06-04 16:27:30.956040227 +0200
@@ -25130,8 +25130,8 @@ rtl8168_start_xmit(struct sk_buff *skb,
         wmb();
         txd->opts1 = cpu_to_le32(opts1);
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(3,5,0)
-        dev->trans_start = jiffies;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,7,0)
+        netif_trans_update(dev);
 #else
         skb_tx_timestamp(skb);
 #endif //LINUX_VERSION_CODE < KERNEL_VERSION(3,5,0)
