--- ./steam	2016-11-24 03:16:35.000000000 +0100
+++ ./steam_b	2016-12-21 23:20:31.275237164 +0100
@@ -3,6 +3,12 @@
 # This is the Steam script that typically resides in /usr/bin
 # It will create the Steam bootstrap if necessary and then launch steam.
 
+# Use alsa for SDL_AUDIODRIVER if there is not pulseaudio installed
+if ! pulseaudio --check &>/dev/null
+    then
+        export SDL_AUDIODRIVER=alsa
+fi
+
 # Get the full name of this script
 export STEAMSCRIPT="$(cd "${0%/*}" && echo "$PWD")/${0##*/}"
 
