--- ./steam	2016-12-21 23:20:31.000000000 +0100
+++ ./steam_b	2016-12-21 23:22:59.798300359 +0100
@@ -3,6 +3,10 @@
 # This is the Steam script that typically resides in /usr/bin
 # It will create the Steam bootstrap if necessary and then launch steam.
 
+# Workaround for dbus fatal termination related coredumps (SIGABRT)
+# https://github.com/ValveSoftware/steam-for-linux/issues/4464
+export DBUS_FATAL_WARNINGS=0
+
 # Use alsa for SDL_AUDIODRIVER if there is not pulseaudio installed
 if ! pulseaudio --check &>/dev/null
     then
