--- ./steam	2016-12-21 23:37:51.489292166 +0100
+++ ./steam_b	2016-12-21 23:39:05.768883208 +0100
@@ -3,6 +3,18 @@
 # This is the Steam script that typically resides in /usr/bin
 # It will create the Steam bootstrap if necessary and then launch steam.

+# Steam runtime settings
+# if STEAM_RUNTIME=1, ignore this entirely
+if ! [ "${STEAM_RUNTIME}" == "1" ]
+    then
+        # if steam-native package is installed, enable native runtime
+        if [ -f "/usr/lib/steam/native_runtime.txt" ]
+            then
+                export STEAM_RUNTIME=0
+                echo "Manjaro steam native configuration found!"
+        fi
+fi
+
 # Workaround for dbus fatal termination related coredumps (SIGABRT)
 # https://github.com/ValveSoftware/steam-for-linux/issues/4464
 export DBUS_FATAL_WARNINGS=0
