--- ./steam	2016-12-21 23:39:05.000000000 +0100
+++ ./steam_b	2016-12-21 23:40:06.365217678 +0100
@@ -3,6 +3,9 @@
 # This is the Steam script that typically resides in /usr/bin
 # It will create the Steam bootstrap if necessary and then launch steam.

+# Set fallback functionality environment variables
+export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/lib32"
+
 # Steam runtime settings
 # if STEAM_RUNTIME=1, ignore this entirely
 if ! [ "${STEAM_RUNTIME}" == "1" ]
