--- i3status.conf.orig	2016-08-20 16:42:02.795964315 +0200
+++ i3status.conf       2016-08-20 16:56:47.165193561 +0200
@@ -9,39 +9,60 @@
 general {
         colors = true
         interval = 5
+        color_good = "#2AA198"
+        color_bad = "#586E75"
+        color_degraded = "#DC322F"
 }
 
-order += "ipv6"
+order += "cpu_usage"
+# order += "load"
 order += "disk /"
+# order += "disk /home"
 order += "wireless _first_"
 order += "ethernet _first_"
-order += "battery 0"
-order += "load"
+order += "battery 1"
+# order += "load"
 order += "tztime local"
+# order += "ipv6"
 
-wireless _first_ {
-        format_up = "W: (%quality at %essid) %ip"
-        format_down = "W: down"
+cpu_usage {
+        format = " cpu  %usage "
 }
 
-ethernet _first_ {
-        # if you use %speed, i3status requires root privileges
-        format_up = "E: %ip (%speed)"
-        format_down = "E: down"
-}
-
-battery 0 {
-        format = "%status %percentage %remaining"
+load {
+        format = " load %1min "
+        # max_threshold = 0.3
 }
 
-tztime local {
-        format = "%Y-%m-%d %H:%M:%S"
+disk "/" {
+        # format = " hdd %avail "
+        format = " ⛁ %avail "
 }
 
-load {
-        format = "%1min"
+ethernet _first_ {
+# if you use %speed, i3status requires root privileges
+        format_up = " lan: %ip %speed "
+        format_down = " no lan "
+}
+
+battery 1 {
+        # format = "%status %percentage %remaining %emptytime"
+        format = " %status %percentage"
+        format_down = "No battery"
+        # last_full_capacity = true
+        integer_battery_capacity = true
+        # status_chr = ""
+        status_chr = "⚡"
+        status_bat = ""
+        # status_bat = "bat"
+        # status_bat = "☉"
+        # status_full = ""
+        status_full = "☻"
+        low_threshold = 15
+        threshold_type = time
 }
 
-disk "/" {
-        format = "%avail"
+tztime local {
+        # format = "%Y-%m-%d %H:%M:%S"
+        format = " %d.%m. %H:%M "
 }
