diff -rupN arch-update/extension.js arch-update-new/extension.js
--- arch-update/extension.js	2017-04-25 07:23:10.000000000 +0200
+++ arch-update-new/extension.js	2017-04-29 18:34:59.028162000 +0200
@@ -38,19 +38,19 @@ const Me = ExtensionUtils.getCurrentExte
 const Utils = Me.imports.utils;
 
 const Format = imports.format;
-const Gettext = imports.gettext.domain('arch-update');
+const Gettext = imports.gettext.domain('manjaro-update');
 const _ = Gettext.gettext;
 
 /* Options */
 let ALWAYS_VISIBLE     = true;
 let SHOW_COUNT         = true;
-let BOOT_WAIT		   = 15;      // 15s
-let CHECK_INTERVAL     = 60*60;   // 1h
+let BOOT_WAIT		   = 30;      // 15s
+let CHECK_INTERVAL     = 360*360;   // 1h
 let NOTIFY             = false;
 let HOWMUCH            = 0;
 let TRANSIENT          = true;
-let UPDATE_CMD         = "gnome-terminal -e 'sh -c  \"sudo pacman -Syu ; echo Done - Press enter to exit; read\" '";
-let CHECK_CMD          = "/usr/bin/checkupdates";
+let UPDATE_CMD         = "update-command";
+let CHECK_CMD          = "/usr/bin/update-check";
 let PACMAN_DIR         = "/var/lib/pacman/local";
 let STRIP_VERSIONS     = true;
 let AUTO_EXPAND_LIST   = 0;
@@ -63,11 +63,11 @@ let UPDATES_LIST       = [];
 
 function init() {
 	String.prototype.format = Format.format;
-	Utils.initTranslations("arch-update");
+	Utils.initTranslations("manjaro-update");
 }
 
-const ArchUpdateIndicator = new Lang.Class({
-	Name: 'ArchUpdateIndicator',
+const ManjaroUpdateIndicator = new Lang.Class({
+	Name: 'ManjaroUpdateIndicator',
 	Extends: PanelMenu.Button,
 
 	_TimeoutId: null,
@@ -78,7 +78,7 @@ const ArchUpdateIndicator = new Lang.Cla
 	_updateList: [],
 
 	_init: function() {
-		this.parent(0.0, "ArchUpdateIndicator");
+		this.parent(0.0, "ManjaroUpdateIndicator");
 		Gtk.IconTheme.get_default().append_search_path(Me.dir.get_child('icons').get_path());
 
 		this.updateIcon = new St.Icon({icon_name: "arch-unknown-symbolic", style_class: 'system-status-icon'});
@@ -285,13 +285,13 @@ const ArchUpdateIndicator = new Lang.Cla
 					if (updateList.length > 0) {
 						// Show notification only if there's new updates
 						this._showNotification(
-							Gettext.ngettext( "New Arch Linux Update", "New Arch Linux Updates", updateList.length ),
+							Gettext.ngettext( "New Manjaro Update", "New Manjaro Updates", updateList.length ),
 							updateList.join(', ')
 						);
 					}
 				} else {
 					this._showNotification(
-						Gettext.ngettext( "New Arch Linux Update", "New Arch Linux Updates", updatesCount ),
+						Gettext.ngettext( "New Manjaro Update", "New Manjaro Updates", updatesCount ),
 						Gettext.ngettext( "There is %d update pending", "There are %d updates pending", updatesCount ).format(updatesCount)
 					);
 				}
@@ -430,13 +430,13 @@ const ArchUpdateIndicator = new Lang.Cla
 
 });
 
-let archupdateindicator;
+let manjaroupdateindicator;
 
 function enable() {
-	archupdateindicator = new ArchUpdateIndicator();
-	Main.panel.addToStatusArea('ArchUpdateIndicator', archupdateindicator);
+	manjaroupdateindicator = new ManjaroUpdateIndicator();
+	Main.panel.addToStatusArea('ManjaroUpdateIndicator', manjaroupdateindicator);
 }
 
 function disable() {
-	archupdateindicator.destroy();
+	manjaroupdateindicator.destroy();
 }
diff -rupN arch-update/metadata.json arch-update-new/metadata.json
--- arch-update/metadata.json	2017-04-25 07:23:10.000000000 +0200
+++ arch-update-new/metadata.json	2017-04-29 18:34:59.041495000 +0200
@@ -1,13 +1,13 @@
 {
-  "description": "Update indicator for Arch Linux and GNOME Shell.\nCan support AUR or other distros by changing command used to check for and apply updates.", 
-  "name": "Arch Linux Updates Indicator",
+  "description": "Update indicator for Manjaro Linux and GNOME Shell.\nSupports AUR and integrates with pamac and mhwd.", 
+  "name": "Manjaro Linux Updates Indicator",
   "shell-version": [
     "3.18",
     "3.20",
     "3.22",
     "3.24"
   ], 
-  "url": "https://github.com/RaphaelRochet/arch-update", 
-  "uuid": "arch-update@RaphaelRochet", 
-  "version": 21
+  "url": "https://github.com/Chrysostomus/arch-update", 
+  "uuid": "manjaro-update@Chrysostomus", 
+  "version": 20
 }
diff -rupN arch-update/prefs.js arch-update-new/prefs.js
--- arch-update/prefs.js	2017-04-25 07:23:10.000000000 +0200
+++ arch-update-new/prefs.js	2017-04-29 18:34:59.041495000 +0200
@@ -42,7 +42,7 @@ function buildPrefsWidget(){
 	let box = buildable.get_object('prefs_widget');
 
 	let version_label = buildable.get_object('version_info');
-	version_label.set_text('[Arch-update v' + Me.metadata.version.toString() + ']');
+	version_label.set_text('[Manjaro-update v' + Me.metadata.version.toString() + ']');
 
 	// Bind fields to settings
 	settings.bind('boot-wait' , buildable.get_object('field_wait') , 'value' , Gio.SettingsBindFlags.DEFAULT);
diff -rupN arch-update/README.md arch-update-new/README.md
--- arch-update/README.md	2017-04-25 07:23:10.000000000 +0200
+++ arch-update-new/README.md	2017-04-29 18:34:59.041495000 +0200
@@ -6,7 +6,7 @@ Update indicator for Arch Linux and GNOM
 - Optional update count display on panel
 - Optional notification on new updates (defaults to off)
 - Launcher for your favorite update command
-- Comes in English, French, Czech, German, Spanish, Brazilian Portuguese, Italian, Polish, Romanian, Arabic, Slovak, Chinese, Serbian, Norwegian Bokmal, Russian, Persian languages. (Thanks translators !)
+- Comes in English, French, Czech, German, Spanish, Brazilian Portuguese, Italian, Polish, Romanian, Arabic, Slovak, Chinese, Serbian, Norwegian Bokmal, Russian languages. (Thanks translators !)
 
 ## One-click install
 It's on extensions.gnome.org :
@@ -21,10 +21,6 @@ To install, simply download as zip and u
 
 ## Changes
 
-### v21
-- Gnome 3.24
-- New translation : Persian
-
 ### v20
 - Translations updates (German, Spanish)
 
diff -rupN arch-update/schemas/org.gnome.shell.extensions.arch-update.gschema.xml arch-update-new/schemas/org.gnome.shell.extensions.arch-update.gschema.xml
--- arch-update/schemas/org.gnome.shell.extensions.arch-update.gschema.xml	2017-04-25 07:23:10.000000000 +0200
+++ arch-update-new/schemas/org.gnome.shell.extensions.arch-update.gschema.xml	2017-04-30 19:19:42.429961000 +0200
@@ -51,13 +51,13 @@
     </key>
 
     <key name="check-cmd" type="s">
-      <default>"/usr/bin/checkupdates"</default>
+      <default>"/usr/bin/update-check"</default>
       <summary>Command to run to check for updated packages.</summary>
       <description>Command to run to check for updated packages.</description>
     </key>
 
     <key name="update-cmd" type="s">
-      <default>"gnome-terminal -e 'sh -c  \"sudo pacman -Syu ; echo Done - Press enter to exit; read\" '"</default>
+      <default>"update-command"</default>
       <summary>Command to run to update packages.</summary>
       <description>Command to run to update packages.</description>
     </key>
